bin/mail/2016-03-21-teacher-recruitment/send-hoc2015-organizers-real-mails

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# 157303 contacts
+../send-to-mailing-list --start      0 --limit 100000 3-17-recruit-hoc-code-studio ./hoc2015-organizers-deduped.csv
+../send-to-mailing-list --start 100000 --limit 100000 3-17-recruit-hoc-code-studio ./hoc2015-organizers-deduped.csv
```

Comment:
Is there any reason send-to-mailing-list can't be modified to batch in sets of 100K to eliminate the need for doing this? Do we ever use a non-100K batch size?